tests/k8s/1-045_validate_applicationset_image_override/01-add_applicationset_image.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+  labels:
+    example: basic
```

```suggestion
apiVersion: argoproj.io/v1beta1
kind: ArgoCD
metadata:
  name: example-argocd
```
Lets remove the label and use `v1beta` api.